File: api2/Models/Round.cs
namespace api2.Models
{
    public class Round
    {
        public Guid Id { get; set; }
        public string? Name { get; set; }
        public int DisqualifyAmount { get; set; }  = 0;
        public int AnswerTime { get; set; } = 30;
        public int Points { get; set; } = 0;
        public bool IsAdditional { get; set; } = false;
    }
}

File: api/app/Models/Round.php
<?php

namespace App\Models;

use Illuminate\Database\Eloquent\Factories\HasFactory;
use Illuminate\Database\Eloquent\Model;
use Illuminate\Database\Eloquent\Relations\BelongsTo;
use Illuminate\Database\Eloquent\Relations\HasMany;
use Illuminate\Database\Eloquent\Concerns\HasUuids;

class Round extends Model
{
    use HasFactory, HasUuids;

    protected $fillable = [
        'id',
        'title',
        'disqualify_amount',
        'answer_time',
        'points',
        'is_additional',
        'game_id',
    ];

    public function game(): BelongsTo {
        return $this->belongsTo(Game::class);
    }

    public function questions(): HasMany {
        return $this->hasMany(Question::class);
    }
}
